clients/python/example/pingpong/pong/pong.Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.8 AS build
+WORKDIR /python
+COPY . .
+
+RUN pip install .
+
+WORKDIR /app
+
+COPY example/pingpong/pong/pong.py .
```

Comment:
if you are doing a dockerfile in the same folder as the python file, no need to define the whole path `example/pingpong/pong/pong.py`, just doing `COPY pong.py .` should work as long as you pass the proper context in the `Makefile` of the pingpong folder